web-portal/build/utils.js

```diff
@@ -0,0 +1,40 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+
+module.exports.cssLoaders = function(options) {
+  options = options || {}
+
+  const cssLoader = {
+    loader: 'css-loader',
+    options: {
+      sourceMap: options.sourceMap
+    }
+  }
+
+  const postcssLoader = {
+    loader: 'postcss-loader',
+    options: {
+      sourceMap: options.sourceMap
+    }
+  }
+
+  function generateLoaders() {
+    const loaders = [cssLoader]
+    if (options.usePostCSS) loaders.push(postcssLoader)
+    // this is standard Vue-CLI setup, but we're never setting extract to true
+    // We were using <style> injection in production with browserify, and though it
+    // would be nice to break it out into a stylesheet it can wait for the next phase
+    if (options.extract) {
+      return ExtractTextPlugin.extract({
+        use: loaders,
+        fallback: 'vue-style-loader'
+      })
+    } else {
+      return ['vue-style-loader'].concat(loaders)
+    }
+  }
+
+  return {
```

Comment:
why call this twice, won't it give the same thing both times

Reply:
This was copied from the Vue CLI 2 config. I think the return value is passed to code that only uses one of the fields on the returned object.
I may have also removed some code that handled other preprocessors.